lua/mason-registry/glint-language-server/init.lua

```diff
@@ -0,0 +1,16 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "glint-language-server",
+    desc = [[Glint Language server, for Glimmer-flavored JavaScript and TypeScript]],
+    homepage = "https://github.com/typed-ember/glint/tree/main/packages/core",
+    categories = { Pkg.Cat.LSP },
+    languages = {
+        Pkg.Lang.Handlebars,
+        Pkg.Lang.Glimmer,
+        Pkg.Lang["typescript.glimmer"],
+        Pkg.Lang["javascript.glimmer"],
+    },
+    install = npm.packages { "@glint/core", "typescript", bin = { "glint-language-server" } },
```

Comment:
Since the installed package is `@glint/core` which seems to ship more than just a language server , I think we should include the additional `glint` executable in `bin`! I also think the name of the package should be changed to just `glint` to more appropriately map to its underlying package

```suggestion
    install = npm.packages { "@glint/core", "typescript", bin = { "glint", "glint-language-server" } },
```

Reply:
> I also think the name of the package should be changed to just glint to more appropriately map to its underlying package

does this mean that the mappings entry in mason-lsp-config wouldn't be needed?